packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -127,7 +128,7 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
                         </Button>
                       )}
                     />
-                    <Tooltip.Content>{t('notification.actions.read.toolTip')}</Tooltip.Content>
+                    <Tooltip.Content>{t('notification.actions.read.tooltip')}</Tooltip.Content>
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** missing `data-localization`